main/src/ca/uwaterloo/flix/api/lsp/provider/completion/DefCompleter.scala

```diff
@@ -46,7 +46,7 @@ object DefCompleter {
     val isMatch = if (isNamespace)
       decl.sym.toString.startsWith(word)
     else
-      decl.sym.text.startsWith(word)
+      false
```

Comment:
Why this change?

Reply:
Otherwise, DefCompleter will provide `A.bar` for `ba` when there is already `use A.bar;`

Reply:
All possible cases now:
List.ma => DefCompleter
ma (without `use List.map;`) => AutoUseCompleter
ma (with `use List.map`) => LocalScopeCompleter

Reply:
OK- that makes sense.